bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -762,13 +763,13 @@ impl ParseError {
 
         match self {
             InvalidInteger { source, input } if *source.kind() == IntErrorKind::PosOverflow => {
-                write!(f, "{} {} is above limit {}", subject, input, upper_bound)
+                write!(f, "{} {} is above limit {}", subject, input.display_raw(), upper_bound)
             }
             InvalidInteger { source, input } if *source.kind() == IntErrorKind::NegOverflow => {
-                write!(f, "{} {} is below limit {}", subject, input, lower_bound)
+                write!(f, "{} {} is below limit {}", subject, input.display_raw(), lower_bound)
             }
             InvalidInteger { source, input } => {
-                write_err!(f, "failed to parse {} as {}", input, subject; source)
+                write_err!(f, "failed to parse {} as {}", input.display_raw(), subject; source)
```

Comment:
Because it drops `subject` and I thought you'd want to use that - I personally would prefer to get rid of `subject`, duplicate code, and make this whole module's error handling more simple but like the PR title says I was just trying to make some forward progress.